docs/dev/README.md

```diff
@@ -1,80 +1,72 @@
 
 * [Reach out to support@datadoghq.com][5] and tell us what metrics you would like to see from a given source.
 * Implement the integration yourself and submit the code to the [official extras repository][6].
 
-If you want to create a new Check from scratch, start by looking at the [howto][10] documentation.
+## Development basics
 
-## Development guide
-
-Follow these instructions to get a working copy of any check on your local Python environment; this is mostly useful to run tests or for tinkering in general.
+The remainder of this document introduces you to the base knowledge and setup required to start working on your own Integration. Once you're comfortable with this content, you can move on to the [technical specifics of Integration development][10].
 
 ### Prerequisites
 
 * Python 2.7, see [this page][7] for more details.
 
 ### Quickstart
 
-The project comes with a requirements file, pass it to `pip` to install all the dependencies needed to work with any check. From the root of the repo, run:
+The project comes with a requirements file usable by `pip` to install all the dependencies needed to work with any Check. From the root of the repo, run:
 
 ```
 pip install -r requirements-dev.txt
 ```
 
-To work with a specific check you need to install its own dependencies. The easiest way to iterate on a check development is installing the wheel itself in editable mode. For example, if you want to do this for the `disk` check run the following:
+You must install the dependencies of a specific Check in order to work with it. The easiest way to iterate during development is by installing the wheel itself in editable mode. Consider the `disk` Check as an example:
 
 ```
 cd disk && pip install -e .
 ```
 
-To double check everything is working as expected run:
+Verify that everything is working as expected:
 
 ```
 python -c"from datadog_checks.disk import Disk"
 ```
 
-if the commands ends without errors, you're good to go!
+If the command exits without errors, you're good to go!
 
 ### Testing
 
-To run the test suite for a given check, either use `tox`, like:
+Use `tox` to run the test suite for a given Check:
 
 ```
 cd {integration} && tox
 ```
 
-If you updated the test requirements for a check, run `tox --recreate` for changes to be effective.
+If you updated the test requirements for a check, run `tox --recreate` for your changes to become effective.
 
 ### Building
 
-`setup.py` provides the setuptools setup script that will help us package and build the wheel. To learn more about python packaging take a look at [the official python documentation][9]
+`setup.py` provides the setuptools setup script that will help us package and build the wheel. To learn more about Python packaging, take a look at [the official python documentation][9]
 
-Once your setup.py is ready, creating a wheel is a easy as:
+Once your `setup.py` is ready, creating a wheel is one command:
```

Comment:
`is one command` looks a little bit odd to me, Would `create a wheel with one command` be better?